### DISCUSSION TITLE
Worst testing tool ever

### DISCUSSION THREAD

**Discussion by dmohapatra10** [cypress-io/cypress#26550] [General]
- I have created a feature branch in Git.
- Added a feature file and automated a test case in Cypress 
- Pushed the commit to Gitlab 
- Now I have to modify one locator and test if its working fine. However this tool is picking the old locator and not the new one.
- I have tried to comment few lines of code and checked if the changes are being identified by Cypress or not, and it didn't.
- Please let me know if you have the fixes else we may consider Selenium for our testing.

**Comment by emilyrohrbough** [comment]
@dmohapatra10 The title of this discussion does not align with our [Code of Conduct](https://github.com/cypress-io/cypress/blob/develop/CODE_OF_CONDUCT.md). If you feel this is a true statement, you are welcome to use a different testing tool. Please update this title to reflect the question you have.

**Comment by nagash77** [comment]
@dmohapatra10 I'm sorry you are having difficulties while trying to use Cypress.  Please check out our [community chat](https://discord.gg/cypress), it can be helpful for debugging or answering questions on how to use Cypress.
